src/test/suite/breakpoints.test.ts

```diff
@@ -0,0 +1,88 @@
+import * as assert from "assert";
+import * as vscode from "vscode";
+import * as sinon from "sinon";
+import { getCursorlessApi } from "../../util/getExtensionApi";
+import { openNewEditor } from "../openNewEditor";
+
+suite("breakpoints", async function () {
+  this.timeout("100s");
+  this.retries(3);
+
+  teardown(() => {
+    sinon.restore();
+  });
+
+  test("breakpoint harp", breakpointHarp);
+  test("breakpoint token harp", breakpointTokenHarp);
+});
+
+async function breakpointHarp() {
+  const graph = (await getCursorlessApi()).graph!;
+
+  await openNewEditor("  hello");
+
+  await graph.hatTokenMap.addDecorations();
+
+  removeBreakpoints();
```

Comment:
should we toss this line into `teardown` above?

Reply:
It has to be no breakpoints before the test it set upnot tear down in that case